index.html

```diff
@@ -91,6 +91,88 @@ <h2>Statement of Intent</h2>
 
 <section>
 <h2>Code</h2>
+
+	<h3 id="unacceptablebehaviour">
+        <strong>Unacceptable behaviour</strong>
+      </h3>
+
+      <p>Some behaviors run counter to the Code of Ethics and Professional Conduct. This list of unacceptable behaviors does not cover every case. Each person you interact with is unique, and behavior must be assessed on an individual level. Ensuring that your behavior does not have a negative impact is your responsibility.</p>
+
+      <ul>
+        <li>Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness,
+          neurotype, physical appearance, body, age, race, socio-economic status, ethnicity, nationality, language, or religion</li>
+
+        <li>Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health,
+          parenting, drugs, and employment</li>
+
+        <li>Deliberate misgendering or use of ‘dead’ or rejected names</li>
+
+        <li>Gratuitous or off-topic sexual images or behaviour in spaces where they are not appropriate</li>
+
+        <li>Physical contact and simulated physical contact (e.g., textual descriptions like “hug” or “backrub”) without consent
+          or after a request to stop</li>
+
+        <li>Threats of violence</li>
+
+        <li>Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm</li>
+
+        <li>Deliberate intimidation</li>
+
+        <li>Stalking or following</li>
+
+        <li>Harassing photography or recording, including logging online activity for harassment purposes</li>
+
+        <li>Sustained disruption of discussion</li>
+
+        <li>Unwelcome sexual attention</li>
+
+        <li>Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others</li>
+
+        <li>Continued one-on-one communication after requests to cease</li>
+
+        <li>Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect other
+          community members or other vulnerable people from intentional abuse</li>
+
+        <li>Publication of non-harassing private communication without consent by the involved parties</li>
+
+        <li>Use of "dog whistles", coded language used to rally support for hate groups and to intmidate vulnerable groups.</li>
+
+        <li>Microaggressions, which are small comments or questions, either intentional or unintentional, that marginalize people
+          by communicating hostile, derogatory, or negative beliefs. Examples include
+
+
+          <ul>
+            <li>Patronizing language or behaviour
+
+
+              <ul>
+                <li>Pedantic corrections that don’t contribute to the conversation (For example: “Well actually…”)</li>
```

Comment:
There are plenty of ways to be pedantic about technical issues without undermining the person speaking.

The phrase "Well actually..." is a good example because it immediately sets the tone of your comment undermining the person you are speaking to, making them feel inferior and discouraging them from participating further whilst not actually contributing anything to the conversation.

Compare 'Well actually,' with 'I respectfully disagree,'

Reply:
> Compare 'Well actually,' with 'I respectfully disagree,'

I feel differently about these phrases @AdaRoseCannon. Their usage varies and invariably there's additional information in tone, body language etc. Claiming to disagree respectfully is sadly not always a sign of genuine respect.

Reply:
However we seem to be agreeing on the main point, that we do not want to accept interjections that _are_ designed to undermine the person speaking.

We could simply change my reword proposal in https://github.com/w3c/PWETF/pull/37#discussion_r271929743 to:

`Interjections that don't contribute to the conversation and are attempts either to undermine the person speaking or to demonstrate superiority of the person interjecting.`

and leave it to the judgement of those involved to assess whether or not the tone is undermining.

Reply:
I am fine with the proposed rewording except that I do think it's important to include the example of "well actually", which has become a buzz phrase to signal mansplaining. (I apologize if that offends people, but it might open your eyes to the fact that opening with this phrase will make people stop listening.)  See for example https://www.urbandictionary.com/define.php?term=well%20actually.

Reply:
>become a buzz phrase

This is exactly why it is a problem. We should aim for these guidelines to stand for some time, even when that phrase has been replaced two or three times with others, which will probably happen given the fluidity of language, in just a few years.

I may be being over-sensitive, but that urban dictionary link is somewhat offensive to me, because it seems to take language that I might sometimes use and others might misuse and I feel implicated unfairly.

Reply:
I am sorry you feel bad about having used this phrasing without ill intent but Tzviya's point is still valid, and you yourself should probably be wary of using it in the future.

If you begin a speaking with someone using that phrase, especially a woman, they are likely going to feel you are not listening to them and just want to drive some point across. Personally it's been said to me so much by condescending men that now whenever someone begins a sentence with it, I immediately start looking for the exits.

This is not a hard and fast rule but it's a clear red flag for this kind of behaviour.

Reply:
Okay @AdaRoseCannon, this thread isn't about me, and I didn't mean to derail it in that direction. You can be sure I will be _extremely_ self-conscious about using that phrase in the future! And I'm happy to learn one way to avoid alienating people in the future, so thank you for that.

Reply:
Thank you for demonstrating a respectful exchange of ideas! minor tweak to language made